agent/engine/docker_task_engine_linux.go

```diff
@@ -17,10 +17,15 @@
 package engine
 
 import (
+	"context"
 	"time"
 
+	"github.com/cihub/seelog"
```

Comment:
Do you want to use "github.com/aws/amazon-ecs-agent/agent/logger" instead of seelog?